stellar-contract-env-host/src/test.rs

```diff
@@ -976,6 +998,7 @@ fn invoke_cross_contract() -> Result<(), HostError> {
 }
 
 #[cfg(feature = "vm")]
+#[ignore]
```

Comment:
Ignoring this test for now since I need to update the test contract to use the new `vec_new` signature.

Reply:
Do you want me to hold merging this, or good to merge and this is happening in a follow up?

Reply:
Let's merge this. I will fix it as a follow up. The original contract code that generated this test isn't in the sdk repo yet.